wagtail/admin/static_src/wagtailadmin/app/wagtailadmin.entry.js

```diff
@@ -1,7 +1,11 @@
-import { initExplorer } from 'wagtail-client';
+import { initExplorer, Icon, Portal } from 'wagtail-client';
+
+// Expose components as globals for third-party reuse.
+window.wagtail.Icon = Icon;
+window.wagtail.Portal = Portal;
```

Comment:
I'm not completely sure about the naming convention for the long-term if were to expose `utils` or other helpers. It would probably make more sense to incorporate `components` into this.

Reply:
Do you mean as `window.wagtail.components.Icon`, and same for Portal? That seems reasonable, but I also like the idea of mimicking what a `wagtail` package would expose – you would do `import { Icon } from 'wagtail';`, without an intermediate `component` object.

Reply:
Yeah exactly like that 😄 But I'm agreeing with you on the mimicking part for the long-haul 👍

Reply:
👌 done. We'll see about the long haul when there actually is a package 😛